dev/requirements.txt

```diff
@@ -1,3 +1,7 @@
 jira==1.0.3
+numpy==1.11.1
+pep8==1.7.0
 PyGithub==1.26.0
+pylint==1.6.4
+Sphinx==1.4.6
 Unidecode==0.04.19
```

Comment:
Could you add `requests==2.11.1`? The version of pip being installed needs a new versions of the requests library that might not be in the default virtualenv everyone has on their machine (and sadly pip doesn't explicitly depend on this).